verification.py

```diff
@@ -4623,8 +4658,31 @@ def run(self, output_dir, names=None, group=None):
             tests = [test for test in self.tests if test.group == group]
         else:
             tests = self.tests
-        for test in tests:
-            test.run(output_dir)
+
+        with concurrent.futures.ProcessPoolExecutor(max_workers=threads) as executor:
+            pbar = tqdm.tqdm(total=len(tests), desc="Queueing tests", leave=True)
+            futures = []
+            for test in tests:
+                futures.append(executor.submit(test.run(output_dir)))
+                pbar.update(1)
+
+            assert len(futures) == len(tests)
+
+            pbar = tqdm.tqdm(total=len(futures), desc="Running tests", leave=True)
+            for future in concurrent.futures.as_completed(futures):
+                pbar.update(1)
+                logging.debug(f"{future} done")
+
+
+def setup_logging(args):
+    log_level = "ERROR"
+    if args.verbose == 1:
+        log_level = "WARN"
+    elif args.verbose == 2:
+        log_level = "INFO"
+    elif args.verbose > 2:
+        log_level = "DEBUG"
+    daiquiri.setup(level=log_level)
```

Comment:
This isn't quite working right - when I put in ``-v``, there's no output (as this corresponds to WARN). I think we just want two levels, so you can get rid of ERROR. The desired behaviour is 
```
python3 verifcation.py -v
```
prints out the INFO messages (then name of the test that's being run and 
```
python3 verifcation.py -vv
```
prints out the INFO and DEBUG messages

Also, at INFO level the messages from msprime swamp out the messages here, so we want to do something like
```
```
python3 verifcation.py -v
```
    daiquiri.setup(level=log_level)
    msprime_logger = daiquiri.getLogger("msprime")
    msprime_logger.setLevel("WARN")
```
to suppress all the messages coming from msprime.